strongbox-cron/strongbox-cron-api/pom.xml

```diff
@@ -116,7 +116,7 @@
         </dependency>
         <dependency>
             <groupId>org.codehaus.groovy</groupId>
-            <artifactId>groovy-all</artifactId>
+            <artifactId>groovy</artifactId>
```

Comment:
(For my better understanding): What's the difference between `groovy` and `groovy-all`?

Reply:
`groovy-all `contains many groovy modules like `groovy-json, groovy-xml, groovy-console, groovy-testng`whereas `groovy` is just core module.